evm_arithmetization/src/cpu/kernel/asm/hash/blake2/ops.asm

```diff
@@ -1,14 +1,14 @@
 // 64-bit right rotation
 %macro rotr_64(rot)
```

Comment:
Identical modification to the one done for `SHA2`, except this one is a 64-bit rotation instead of 32.